src/main/java/org/zalando/nakadi/service/subscription/state/StreamingState.java

```diff
@@ -441,21 +443,53 @@ private void reconfigureKafkaConsumer(final boolean forceSeek) {
 
         if (!currentAssignment.equals(newAssignment)) {
             try {
-                final List<NakadiCursor> cursors = new ArrayList<>();
-                for (final EventTypePartition pk : newAssignment) {
-                    // Next 2 lines checks that current cursor is still available in storage
-                    final NakadiCursor beforeFirstAvailable = getBeforeFirstCursor(pk);
```

Comment:
method getBeforeFirstCursor(pk)  should be deleted

Reply:
done 36019cb